### PR TITLE
LMA-260

### PR DESCRIPTION
**Pull request by SandraPG85**
…ains

#### Things to review:
- [ ] Unit tests
- [ ] No hardcoded resources (strings, sizes, colors...)
- [ ] Sorted usings
- [ ] Add summaries where needed 

If needed, you can add new checkboxes to https://github.com/fluendo/VAS/blob/master/.github/PULL_REQUEST_TEMPLATE.md

### CODE REVIEW

File: VAS.Drawing.Cairo/CairoBackend.cs [hunk 1]
```diff
@@ -564,16 +560,21 @@ public Image Copy (ICanvas canvas, Area area)
 			Pixmap pm;
 			global::Cairo.Context ctx;
 
+			var w = canvas.Widget;
+			NoWindowWidget widget = new NoWindowWidget { Width = area.Width, Height = area.Height };
+			canvas.SetWidget (widget);
```
Comment (vmartos) [inline VAS.Drawing.Cairo/CairoBackend.cs]: Why this code?

File: VAS.UI.Gtk2/UI/TreeView/CellRendererImage.cs [hunk 2]
```diff
@@ -21,15 +21,21 @@
 using VAS.Core.Common;
 using VAS.Core.Interfaces.Drawing;
 using VAS.Drawing.Cairo;
+using VAS.Drawing.Widgets;
 using Image = VAS.Core.Common.Image;
 using Point = VAS.Core.Common.Point;
+using Color = VAS.Core.Common.Color;
+using VAS.Services;
```
Comment (vmartos) [inline VAS.UI.Gtk2/UI/TreeView/CellRendererImage.cs]: sort usings

File: VAS.UI.Gtk2/UI/TreeView/CellRendererImage.cs [hunk 3]
```diff
@@ -43,6 +49,10 @@ public CellRendererImage (Image image)
 			set;
 		}
 
+		[GLib.Property ("MaskColor")]
+		public Color MaskColor { get;
```
Comment (vmartos) [inline VAS.UI.Gtk2/UI/TreeView/CellRendererImage.cs]: format propertly MaskColor {
get;

File: data/icons/hicolor/scalable/actions/vas-default-shield.svg [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    </defs>
+    <g id="Symbols" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```
Comment (vmartos) [inline data/icons/hicolor/scalable/actions/vas-default-shield.svg]: Remember to pass the script to change/ add vas-default-shield@2x

File: VAS.Drawing.Cairo/CairoBackend.cs (hunk 1 above)
Comment (SandraPG85) [reply VAS.Drawing.Cairo/CairoBackend.cs]: there was a problem in the preview calculation of teams when it was generated by the desktop application. This code is the fix, the problem was an extra scaling, setting already the widget with the proper size fix it.

**Pull request merged**